app/src/main/java/net/squanchy/service/firebase/FirebaseDbService.java

```diff
@@ -37,8 +38,10 @@
     private static final String TRACKS_NODE = "data/tracks";
     private static final String VENUE_INFO_NODE = "data/venue";
     private static final String TRACKS_BY_ID_NODE = "data/tracks/%1$s";
-    private static final String FAVORITES_NODE = "user/%1$s/";
-    private static final String FAVORITES_BY_ID_NODE = "user/%1$s/favorites/%2$s";
+    private static final String FAVORITES_NODE = "user/%1$s/favorites";
+    private static final String FAVORITES_BY_ID_NODE = "user/%1$s/favorites/map/%2$s";
```

Comment:
the new structure for data is
`{
  "data": { /* blah blah */ },
  "user": {
    "any_user_id": {
      "favorites": {
        "event_id_1": true,
        "event_id_4": true,
        "event_id_56": true,
        "event_id_98": true
      },
      "achievements": {
        "achievement_id_1": 12364546798,
        "achievement_id_2": 12364745700
      }
    }
  }
}`